### DISCUSSION TITLE
Idea : Upload progress bar

### DISCUSSION THREAD

**Discussion by Ikarias** [ihexxa/quickshare#213] [Ideas]
As the title says. 

Im uploading a bunch of photos and a few video's, but there is no progress bar, which makes it a guess how far she is.

**Comment by ihexxa** [comment]
Currently the app starts uploading one by one according to the created time order instead of concurrently, then user may be not able to see the one in uploading, which is a bit confusing and could be improved, thanks for the feedback.
